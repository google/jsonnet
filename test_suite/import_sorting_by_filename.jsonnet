// We sort by filename, variable name doesn't matter
local a = import 'z.jsonnet';
local b = import 'y.jsonnet';
local c = import 'x.jsonnet';
true
