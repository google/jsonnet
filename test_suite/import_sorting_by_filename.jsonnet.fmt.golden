// We sort by filename, variable name doesn't matter
local c = import 'x.jsonnet';
local b = import 'y.jsonnet';
local a = import 'z.jsonnet';
true
